data/json/items/gun/500.json

```
@@ -8,6 +8,7 @@
     "description": "Built in the image of the venerable Winchester Model 1886, Big Horn Armory's Model 89 rifle packs all the power of the .500 S&W Magnum cartridge in to a package less likely to break one's wrist.",
     "weight": "3447 g",
     "volume": "3 L",
+    "longest_side": "940 mm",
```

```suggestion
    "volume": "1995 ml",
    "longest_side": "953 mm",
```
Listing of the gun with 18" configuration as you chose for the longest side with views on all sides: https://www.gunsinternational.com/guns-for-sale-online/rifles/big-horn-armory/big-horn-armory---89----500-s-w.cfm?gun_id=101297785#lg-1
Extraction of gun from image for scaling: https://imgur.com/a/HAU8mtY
SA+Feret: https://imgur.com/a/3IxR2ni
Measuring width of grip cap to match to the bottom view of the rifle to get width: https://imgur.com/a/wERhTUn
Width after scaling to grip cap's width from bottom view of gun: https://imgur.com/a/plqdfrx